plugins/modules/mso_schema_template_external_epg_selector.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Cindy Zhao (@cizhao) <cizhao@cisco.com>
```

Comment:
You can leave Cindy's name but don't forget to add your own ;) It is your work too.